stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/MessageComposer.kt

```diff
@@ -226,11 +229,33 @@ public fun MessageComposer(
             onCommandSelected = onCommandSelected
         )
     },
-    shouldShowIntegrations: Boolean = true,
-    integrations: @Composable RowScope.(MessageComposerState) -> Unit,
-    input: @Composable RowScope.(MessageComposerState) -> Unit,
+    integrations: @Composable RowScope.(MessageComposerState) -> Unit = {
+        DefaultComposerIntegrations(
+            messageInputState = it,
+            onAttachmentsClick = onAttachmentsClick,
+            onCommandsClick = onCommandsClick
+        )
+    },
+    label: @Composable () -> Unit = { DefaultComposerLabel() },
+    input: @Composable RowScope.(MessageComposerState) -> Unit = {
+        DefaultComposeInputContent(
+            messageComposerState = messageComposerState,
+            onValueChange = onValueChange,
+            onAttachmentRemoved = onAttachmentRemoved,
+            label = label,
+        )
+    },
+    trailingContent: @Composable (MessageComposerState) -> Unit = {
+        DefaultMessageComposerTrailingContent(
+            value = it.inputValue,
+            cooldownTimer = it.cooldownTimer,
```

Comment:
Would it be better to name this `cooldownTime` instead of `cooldownTimer`?

`cooldownTimer` seems to imply functionality when this is only an integer that represents how much time is left until the user can send a message again.